### PR TITLE
Add displaying the waiting for a payment page

### LINKED ISSUE #25: Create a `Waiting for a payment` page
Currently, when a BLIK code is provided, we are redirected directly to the `order's show page`, so we see a list of available payment methods. Not handling the `processing` payment state is what causes it. This task aims to provide a `Waiting for a payment page`, which will be displayed for every payment that is in processing.

## Small wins 🏆

- [x] After providing a BLIK token, the customer is redirected to the waiting for a payment page
- [x] This page is refreshed automatically, so the customer can just wait for the status update
- [x] Once a payment becomes completed, this page should redirect the customer to the thank you page
- [ ] After `<param>` minutes, if the payment is not completed, the customer should be redirected to a payment failure page

### PR DESCRIPTION
**Pull request by jakubtobiasz**
Closes #25

### CODE REVIEW

File: src/Route.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommerceWeavers\SyliusTpayPlugin;
+
+final class Route
```
Comment (lchrusciel) [inline src/Route.php:7]:
```suggestion
final class Routing
```

File: src/Controller/DisplayWaitingForPaymentPage.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    {
+        $token = $this->payum->getHttpRequestVerifier()->verify($request);
+
+        $resolveNextRoute = $this->resolveNextRouteFactory->createNewWithModel($token);
+
+        $this->payum->getGateway($token->getGatewayName())->execute($resolveNextRoute);
+
+        if (
+            is_string($resolveNextRoute->getRouteName()) &&
+            $resolveNextRoute->getRouteName() !== 'commerce_weavers_tpay_waiting_for_payment'
```
Comment (lchrusciel) [inline src/Controller/DisplayWaitingForPaymentPage.php:36]: once we have route class, would be nice to use it here 
```suggestion
            $resolveNextRoute->getRouteName() !== 'Route::SHOP_WAITING_FOR_PAYMENT'
```

**Review by lchrusciel — APPROVED** [review]

File: config/config.php [hunk 3]
```diff
@@ -6,4 +6,8 @@
 
 return function(ContainerConfigurator $container): void {
     $container->import('config/**/*.php');
+
+    $parameters = $container->parameters();
+
+    $parameters->set('commerce_weavers_tpay.waiting_for_payment.refresh_interval', 5);
```
[the remark this answers is not in the document]
Comment (jakubtobiasz) [reply config/config.php:12]: https://github.com/CommerceWeavers/SyliusTpayPlugin/issues/54

File: src/Route.php (hunk 1 above)
Comment (jakubtobiasz) [reply src/Route.php:7]: IMO `Route::SHOP_WAITING_FOR_PAYMENT > Routing::SHOP_WAITING_FOR_PAYMENT > Routing::SHOP_WAITING_FOR_PAYMENT_ROUTE`

**Pull request merged**